### DISCUSSION TITLE
Major Feature Backlog and Status

### DISCUSSION THREAD

**Discussion by carTloyal123** [jfarmer08/homebridge-wyze-smart-home#224] [Ideas]
This thread is to keep track of the requested community features that are long standing and/or will require large engineering effort to complete. If you have a feature suggestion feel free to put it in this thread as issues requesting features will be catalogued here instead of staying as long standing open issues.

- [ ] Video Streaming Support for Wyze Cameras (will need a topic for each class of camera) #191 
- [ ] Full garage door support with safety #193 
- [x] Wyze Lock Bolt (YD_BT1) #181 
- [ ] Wyze Air Purifier #178 
- [ ] Adaptive Lighting in Homekit #81

**Comment by tcatlas** [comment]
Was #81 (and others listed) actually completed and shipped? I don’t think issues should be closed unless they are completed as they allow for discussion in their own “thread” and thus organized instead of being pooled here. 

I do think keeping this discussion for overall tracking is a great idea though.

**Reply by jfarmer08** [reply]
@DiscoveryOV I am good with what ever you guys decide.

**Comment by Zamua** [comment]
i have a few wyze purifiers, and the garage door controller; can i aid in development? are there any suggested onboarding docs or am i better off just reading the code?

also this plugin is awesome thanks so much

**Comment by jfarmer08** [comment]
There are not onboarding documents at this time, but I would be glad to help with any questions. 

Any addon feature go into this repo and for any api related goes to https://github.com/jfarmer08/wyze-api

**Reply by dlewanda** [reply]
I am also interested in helping with development, specifically in service to improving the garage door functionality. It's been a number of years since I did Node.js development, however. Where would I start after cloning the repo? Is there a way to point my local Homebridge instance at my development version without having to fork?

**Comment by jeremypetz** [comment]
Is the doorbell cam v2 supported?

**Reply by Grapplerone** [reply]
I'm also looking for support for the Wyze v2 doorbell. I don't see how to integrate it into this. If I could get it into homekit and see the doorbell presses there I could do SO MUCH MORE!
